kak-lsp.toml

```diff
@@ -4,20 +4,37 @@ verbosity = 2
 # Semantic tokens support
 # See https://github.com/microsoft/vscode-languageserver-node/blob/8c8981eb4fb6adec27bf1bb5390a0f8f7df2899e/client/src/semanticTokens.proposed.ts#L288
 # for token/modifier types.
+[[semantic_tokens]]
+name = "type"
+face = "type"
 
-[semantic_tokens]
-type = "type"
-variable = "variable"
-namespace = "module"
-function = "function"
-string = "string"
-keyword = "keyword"
-operator = "operator"
-comment = "comment"
-
-[semantic_token_modifiers]
-documentation = "documentation"
-readonly = "default+d"
+[[semantic_tokens]]
+name = "variable"
+face = "variable"
+
+[[semantic_tokens]]
+name = "namespace"
+face = "module"
+
+[[semantic_tokens]]
+name = "function"
+face = "function"
+
+[[semantic_tokens]]
+name = "string"
+face = "string"
+
+[[semantic_tokens]]
+name = "keyword"
+face = "keyword"
+
+[[semantic_tokens]]
+name = "operator"
+face = "operator"
+
+[[semantic_tokens]]
+name = "comment"
+face = "comment"
```

Comment:
Could you also sort them alphabetically?

Reply:
I could, but given that the sort order is tied to functionality here, it might actually be counter-productive to have the reference/example be sorted based on aesthetics.

Reply:
AIUI sorting by token name doesn't have an impact with the current behavior (only sorting by modifier does).
A larger config probably wants to group the config by token type, to make the order obvious.

We can keep it of course. I just like to sort these because it makes it easier search, and easiei to decide where to add new entries